runtime/storage/in-memory-storage.js

```diff
@@ -135,7 +153,7 @@ class InMemoryCollection extends InMemoryStorageProvider {
       // TODO: batch by store.
       let retrieveItem = async item => {
         let ref = item.value;
-        let store = await this._storageEngine.connect(referredType.toString(), referredType, ref.storageKey);
+        let store = await this._storageEngine.share(referredType.toString(), referredType, ref.storageKey);
         return await store.get(ref.id);
```

Comment:
Not related to this CL, but I just noticed: since the outputs of retrieveItem are awaited with a Promise.all, shouldn't just return the store.get Promise directly? I.e. no need to await the get here...

Reply:
Hmm, I think you're right. Will fix.